templates/default/st2.conf.erb

```diff
@@ -65,12 +84,6 @@ remote_dir = /tmp
 [action_sensor]
 triggers_base_url = <%= @api_url %>/v1/triggertypes/
 
-[resultstracker]
-logging = <%= @conf_root %>/st2actions/console.conf
-
-[notifier]
-logging = <%= @conf_root %>/st2actions/console.conf
-
 [database]
```

Comment:
You can remove this. https://github.com/StackStorm/st2/blob/master/conf/st2.prod.conf. 

If not provided, default host is 127.0.0.1 and port to connect would be 27017.